static/stylesheets/theme_us.css

```
@@ -210,11 +219,24 @@
     top: 98px;
 }
 
+/* 古いiPhone向けの設定 ************************************************/
+    @media screen and (min-width:320px) and (max-width:428px){
+      .font-ja .welcome_message {
```

```suggestion
      .welcome_message:lang(ja) {
```
日本語だけスタイルを効かせるには、これでもいいのかなと思いました。